source/views/faqs/index.js

```diff
@@ -1,5 +1,5 @@
 // @flow
-import * as React from 'react'
+import React, {useState, useEffect} from 'react'
```

Comment:
We typically do `* as React`, because there are certain things that Flow only puts on the React object if you do, and that prevents us from spending ten minutes diving into the Flow internals.